### DISCUSSION TITLE
[Resolved] TTGO LoRa 32 V2.1 _ 1.6

### DISCUSSION THREAD

**Discussion by icicmoi** [SQ9MDD/TTGO-T-Beam-LoRa-APRS#43] [Q&A]
Hi all,
and first of all, I hope I don't make a mistake in opening a discussion here Rysiek?

I can't install the correct version on my TTGO LoRa V2.1_1.6 module :
```
esptool.py --chip esp32 --baud 115200 --port /dev/ttyUSB0 write_flash -fm dio --flash_size detect 0x00000 ttgoo-lora32-vxxx.bin 
Response is : Wrote 918800 bytes (538909 compressed) at 0x00000000 in 47.5 seconds (effective 154.8 kbit/s)...
Hash of data verified.
Leaving...
Hard resetting via RTS pin...
```
But nothing on display, wifi or bluetooth .
( and tried these 3 files : ttgo-lora32-v1-2021-07-21T092411.bin, ttgo-lora32-v2-2021-07-21T092411.bin and ttgo-lora32-v2.1-2021-07-21T092411.bin )

So I considered downloading the source file but it seems suitable for the "T-beam" module only : how should I go about it?
thanks a lot for your help :-)

73.
Philippe, f5rvx

**Comment by SQ9MDD** [comment]
Hi Philippe
Display is disabled by default, have you tried to configure it via web after uploading ?

**Reply by icicmoi** [reply]
Thank you, Rysiek, for this information about the display.
But the problem seems in the binary file (which would not be suitable for my ttgo module?) : no information in the terminal and no wifi hotspot either. 
73, Philippe
ps : on my monitor (minicom) I have this error :

rst:0x10 (RTCWDT_RTC_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
flash read err, 1000
ets_main.c 371

**Reply by icicmoi** [reply]
Problem solved or more exactly worked around by using, as you will suggest in the Readme.me, Esphome-flasher  :-)
